.github/workflows/validate-linux-binaries.yml

```
@@ -9,6 +9,7 @@ on:
   pull_request:
     paths:
       - .github/workflows/validate-linux-binaries.yml
+      - .test/smoke_test/smoke_test.py
```

I guess if you change the image you want to re-run this workflow as well?
```suggestion
      - .test/smoke_test/*
```